app/views/admin/_email.html.erb

```diff
@@ -55,6 +55,7 @@
         <%= link_to '#', id: 'queue-test-email-btn', class: 'btn btn-default' do %>
           <%= image("test", title: "Send testing email", alt: "Test")%> Queue test email
         <% end %>
+        <p class="help-block">Note that testing the Queue requires settings to be saved first</p>
```

Comment:
Perhaps in the future we could make the custom job take an optional hash of SMTP settings to temporarily override the saved settings, so it can be used for testing without needing to save the settings first.